# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/mlgpu/RenderViewMLGPU.cpp
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 gfx/layers/mlgpu/RenderViewMLGPU.cpp
--- a/gfx/layers/mlgpu/RenderViewMLGPU.cpp	Sun May 26 14:29:56 2019 +0000
+++ b/gfx/layers/mlgpu/RenderViewMLGPU.cpp	Sun May 26 14:30:14 2019 +0000
@@ -7,7 +7,7 @@
 #include "RenderViewMLGPU.h"
 #include "ContainerLayerMLGPU.h"
 #include "FrameBuilder.h"
-#include "gfxPrefs.h"
+#include "mozilla/StaticPrefs.h"
 #include "LayersHelpers.h"
 #include "LayersLogging.h"
 #include "MLGDevice.h"
@@ -74,7 +74,7 @@
       mCurrentMaskRectBufferIndex(kInvalidResourceIndex),
       mCurrentDepthMode(MLGDepthTestMode::Disabled),
       mNextSortIndex(1),
-      mUseDepthBuffer(gfxPrefs::AdvancedLayersEnableDepthBuffer()),
+      mUseDepthBuffer(StaticPrefs::AdvancedLayersEnableDepthBuffer()),
       mDepthBufferNeedsClear(false) {
   if (aParent) {
     aParent->AddChild(this);
@@ -188,7 +188,7 @@
   // CPU-based occlusion culling as well, since the GPU will do most of our
   // culling work for us.
   if (mUseDepthBuffer || !aItem.translation ||
-      !gfxPrefs::AdvancedLayersEnableCPUOcclusion()) {
+      !StaticPrefs::AdvancedLayersEnableCPUOcclusion()) {
     // Update the render region even if we won't compute visibility, since some
     // layer types (like Canvas and Image) need to have the visible region
     // clamped.